src/main/java/io/github/seggan/sfcalc/StringRegistry.java

```diff
@@ -25,18 +27,23 @@
     private final String notAPlayerString;
     private final String invalidNumberString;
 
-    StringRegistry() {
-        headerString = SFCalc.inst().getConfig().getString("header-string", "&e&nRecipe for %s:");
-        headerAmountString = SFCalc.inst().getConfig().getString("header-amount-string", "&e&nRecipe for %d %s:");
-        stackString = SFCalc.inst().getConfig().getString("stack-string", "&e%d (%d x%d + %d)");
-        amountString = SFCalc.inst().getConfig().getString("amount-string", "&e%s of %s");
-        neededString = SFCalc.inst().getConfig().getString("needed-string", "&e%s more %s needed");
-        noItemString = SFCalc.inst().getConfig().getString("no-item-string", "&cThat item was not found.");
-        notANumberString = SFCalc.inst().getConfig().getString("not-a-number-string", "&cThat's not a number!");
-        tooManyCategoriesString = SFCalc.inst().getConfig().getString("category-error-string", "&cThat many categories is not supported yet. Please use the command form of the calculator.");
-        tooManyItemsString = SFCalc.inst().getConfig().getString("item-error-string", "&cThat many items is not supported yet. Please use the command form of the calculator.");
-        notAPlayerString = SFCalc.inst().getConfig().getString("not-a-player-string", "&cYou must be a player to send this message!");
-        invalidNumberString = SFCalc.inst().getConfig().getString("invalid-number-string", "&cInvalid number!");
+    StringRegistry(AddonConfig config) {
+        if (config.getString("header-string").contains("%s")) {
+            // The config is outdated, overwrite it with default values
+            config.resetToDefaults();
```

Comment:
Oh god, please don't just forcefully reset things. You should never force shit for users.

Reply:
agree

Reply:
the plugin just doesnt work without resetting it...

Reply:
i need to monitor that. i havent gotten any complaints in a while so its safe to assume that most servers have switched and we dont need that